test_only.go

```diff
@@ -0,0 +1,97 @@
+/*
```

Comment:
we could rename this file to test_extensions.go

Reply:
Done.